src/mol-io/writer/cif.ts

```diff
@@ -53,5 +53,58 @@ export namespace CifWriter {
                 return ff && ff.binaryEncoding ? ArrayEncoder.fromEncoding(ff.binaryEncoding) : void 0;
             }
         }
+    };
+
+    export function createEncodingProviderFromJsonConfig(hints: EncodingStrategyHint[]): EncodingProvider {
+        return {
+            get(c, f) {
+                for (let i = 0; i < hints.length; i++) {
+                    const hint = hints[i];
+                    if (hint.categoryName === c && hint.columnName === f) {
+                        return resolveEncoding(hint);
+                    }
+                }
+            }
+        }
     }
+
+    function resolveEncoding(hint: EncodingStrategyHint): ArrayEncoder | undefined {
```

Comment:
You can add

```
type EncodingType = 'pack' | 'rle' | 'delta' | 'delta-rle'
```

and then you can return `ArrayEncoder` because via the switch statement the compiler will check that all cases of the union type are handled.

Reply:
Addressed in 
df6b163